jekyll/_cci2/templates/template-how-to.adoc

```diff
@@ -0,0 +1,179 @@
+
+* Develop new content quickly
+* Ensure your page conforms to the style guide
+
+To use this template, make a copy and place it in the `_cci2` directory. The filename should match the page title. Look at the source file for this page template link:https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/template/template-how-to.adoc?plain=1[here]. There is additional detail in the comments.
+
+The opening paragraph should be used to briefly describe to the reader what the reader will do using this guide, and what the outcomes are when they successfully finish the task.
+
+Contributors from within CircleCI can also consider setting up a local development environment to preview changes before pushing. Ping one of the Developer Resources and Engineering team (DRE) for more information.
+
+[#introduction]
+== Introduction
+
+You may use the introduction to tell the reader what value the feature/task provides, and what use cases it supports. If possible, display the most relevant or popular use cases in a short bullet list. 
+
+As a guideline, the introduction should try to answer the following questions for the reader:
+
+* What’s in it for me?
+* How does this feature / task help me create more value?
+
+Consider providing a link to an xref:template-conceptual#[overview or conceptual doc], should the reader be interested in more explanatory or background information regarding the feature or task.
+
+Since the reader will usually want to quickly get to the task at hand, keep the introduction succinct.
+
+[#prerequisites]
+== Prerequisites
+
+List any prerequisites that the reader needs to have already met in order for them to successfully complete the task. These may include:
+
+// The following will render as an unordered (bullet) list
+
+* Completing a certain task, such as signing up for a CircleCI account and having at least one project building in CircleCI
+* Roles and permissions
+* Knowledge of or familiarity with a certain topic
+// This renders as a nested item
+** Link to any helpful documentation or pages that they can refer to, for example, "For more information on how test splitting works, refer to the xref:https://circleci.com/docs/parallelism-faster-jobs#[Test splitting and parallelism] page."
+* CircleCI plan
+
+Sometimes, there may be topics that are helpful to know, or tasks to complete, that are _helpful_ in order to successfully finish the how-to guide, but do not necessarily have a severely negative impact on the user's flow if they do those as they go along. For example, you may include generating an API token in the how-to steps itself, rather than listing it as a strict prerequisite. Use your best judgment to decide whether to add such a requirement as a prerequisite or as a step that is part of the how-to.
+
+// The section headings in which you outline the steps should be in an active voice
+[#carry-out-the-task]
```

Comment:
@rosieyohannan I like how you did it in the tutorial template here https://github.com/circleci/circleci-docs/pull/8321/files#diff-b6b45f7729d232230111850cfe437bb0a121901fbcd88d0a0dd13c3074507584R60 If it sounds good to you I'll update this to match

Reply:
Yes sounds good to me ✅

Reply:
I wonder whether in the how-to guide template we should give the option of having a few short how-tos on a page, like we already have for a few subjects (so several main use case headings at level 2 with an ordered list to outline steps): https://circleci.com/docs/test-config-policies/ and https://circleci.com/docs/rotate-project-ssh-keys/ for example